js/cryptocom.js

```diff
@@ -0,0 +1,1037 @@
+        const message = this.safeString (response, 'message');
+        if (errorCode in this.exceptions['exact']) {
+            const Exception = this.exceptions[errorCode];
+            throw new Exception (this.id + ' ' + message);
+        }
+        if (errorCode !== '0') {
+            throw new ExchangeError (this.id + ' ' + message);
+        }
+    }
+
+    parseTicker (ticker, market = undefined) {
+        // {
+        //     "i":"CRO_BTC",
+        //     "b":0.00000890,
+        //     "k":0.00001179,
+        //     "a":0.00001042,
+        //     "t":1591770793901,
+        //     "v":14905879.59,
+        //     "h":0.00,
+        //     "l":0.00,
+        //     "c":0.00
+        // }
+        let timestamp = this.safeInteger (ticker, 't');
+        if (timestamp === undefined) {
+            timestamp = this.milliseconds ();
+        }
+        let symbol = undefined;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+        }
+        let open = undefined;
+        let change = undefined;
+        let average = undefined;
+        const last = this.safeNumber (ticker, 'a');
+        const relativeChange = this.safeNumber (ticker, 'c');
+        if (relativeChange !== -1) {
+            open = last / this.sum (1, relativeChange);
+            change = last - open;
+            average = this.sum (last, open) / 2;
+        }
+        return {
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeNumber (ticker, 'h'),
+            'low': this.safeNumber (ticker, 'l'),
+            'bid': this.safeNumber (ticker, 'b'),
+            'bidVolume': undefined,
+            'ask': this.safeNumber (ticker, 'k'),
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': open,
+            'close': last,
+            'last': last,
+            'previousClose': undefined,
+            'change': change,
+            'percentage': relativeChange * 100,
+            'average': average,
+            'baseVolume': this.safeNumber (ticker, 'v'),
+            'quoteVolume': undefined,
+            'info': ticker,
+        };
+    }
+
+    parseTrade (trade, market = undefined) {
+        const timestamp = this.safeInteger (trade, 't');
+        let symbol = undefined;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+        }
+        const price = this.safeNumber (trade, 'p');
+        const amount = this.safeNumber (trade, 'q');
+        const cost = price * amount;
+        const side = this.safeStringLower (trade, 's');
+        const id = this.safeString (trade, 'd');
+        const type = 'limit'; // all trades are still limit trades
+        const takerOrMaker = undefined; // can't distinguish my deal from someone else's
+        const order = undefined; // can't distinguish my deal from someone else's
+        const fee = undefined;
```

Comment:
This handles only public/get-trades, but private/get-trades has different result properties:
```
      {
        "side": "SELL",
        "instrument_name": "ETH_CRO",
        "fee": 0.014,
        "trade_id": "367107655537806900",
        "create_time": 1588777459755,
        "traded_price": 7,
        "traded_quantity": 1,
        "fee_currency": "CRO",
        "order_id": "367107623521528450"
      }
```